js/battle.js

```diff
@@ -5331,6 +5331,7 @@ var Battle = (function () {
 		case 'c':
 			name = args[1];
 			if (this.ignoreSpects && (name.charAt(0) === ' ' || name.charAt(0) === '+')) break;
+			if (this.ignoreOpponent && this.getSide(name) === this.yourSide) break;
```

Comment:
Updated. Not sure what to do with <code>/addplayer</code>, but I don't think we'll see very many situations where someone has both their opponent ignored and the need to replace them.